libs/libxx/libcxx-misc.patch

```diff
@@ -0,0 +1,22 @@
+--- libcxx/src/locale.cpp.orig4	2020-10-23 07:25:28.000000000 +0900
++++ libcxx/src/locale.cpp	2020-10-23 07:25:38.000000000 +0900
```

Comment:
let's merge the patch into one?

Reply:
i suspect it's better to keep things separate for making them upstream.
this patch is separated because it isn't directly related to nuttx.

Reply:
How about libcxx-nuttx-macos.patch? it should merge into 0001-libcxx-Port-to-NuttX-https-nuttx.apache.org-RTOS.patch.

Reply:
i was assuming that 0001-libcxx-Port-to-NuttX-https-nuttx.apache.org-RTOS.patch was already "in-flight". isn't it the case? if it isn't, i have no problem to merge.

Reply:
Not merge yet:
https://reviews.llvm.org/D88718
I can update the PR with your change.

Reply:
do they require Signed-off-by?

Reply:
i submitted libcxx-misc.patch https://reviews.llvm.org/D90139

i will try to merge libcxx-nuttx-macos.patch into 0001-libcxx-Port-to-NuttX-https-nuttx.apache.org-RTOS.patch later.

Reply:
i merged patches in this PR.
if you prefer git: https://github.com/llvm/llvm-project/compare/llvmorg-11.0.0...yamt:nuttx-patches
@xiaoxiang781216 if it looks ok to you, please update https://reviews.llvm.org/D88718

Reply:
Sure, I will update PR tonight.

Reply:
@yamt https://reviews.llvm.org/D88718 is updated and https://github.com/apache/incubator-nuttx/pull/2243 for nuttx